README.md

```diff
@@ -1,131 +1,13 @@
-Chitter Challenge
-=================
+You've come to Dash's Chitter! Good effort!
 
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+This web app will allow you to say what's on your mind (as long as it's no longer than 200 characters obviosuly!) so that all your friends will be able to see what you are thinking and what you are up to!
 
-Challenge:
--------
+To run, have PG installed as a gem.
 
-As usual please start by forking this repo.
+If you need to set up the database from scratch do the following:
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+- Connect to psql
```

Comment:
Good clear instructions, allowed me to set up the required database and table in seconds